api/campaign.js

```diff
@@ -0,0 +1,14 @@
+import { fetchSingleEntry } from './contentfulService'
+import { unwrapFields, unwrapImages, unwrapTeamMember } from './common'
+
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Why not disable the linter rule instead?

Reply:
I will do it.